parsec/backend/postgresql/handler.py

```diff
@@ -25,6 +25,8 @@
 
 CREATE_MIGRATION_TABLE_ID = 2
 MIGRATION_FILE_PATTERN = r"^(?P<id>\d{4})_(?P<name>\w*).sql$"
+# We use int to avoid float error
```

Comment:
should add a comment to explain why this is needed:

```
# Internally, PostgreSQL stores timestamps as 4bytes floating number representing the number of seconds since
# 2000-01-01T00:00:00Z, hence we need to do back-and-forth conversion with the regular 1970-based POSIX epoch
```